db/migrate/20200114125900_update_imported_from_whitehall.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class UpdateImportedFromWhitehall < ActiveRecord::Migration[6.0]
+  def change
+    TimelineEntry.where(entry_type: :imported_from_whitehall).each do |timeline_entry|
```

Comment:
Note to self:
```suggestion
    TimelineEntry.imported_from_whitehall.each do |timeline_entry|
```

Reply:
Somewhat moot given my last comment but you'd not want to do that as that change wouldn't work once https://github.com/alphagov/content-publisher/pull/1644/files#diff-73d3559ae89c6d594ff5fb797014be76L61 is applied.